package.json

```diff
@@ -5,7 +5,8 @@
   "license": "MIT",
   "workspaces": {
     "packages": [
-      "packages/*"
+      "packages/*",
+      "playground/elements/*"
```

Comment:
I'm not in favor of putting playground stuff into the workspace. This will only get annoying with possible multiple versions of the same dependency and bloat the install size even if you don't want to run it.

You should use yalc or secco to link the package to the playground IMO

Reply:
@LekoArts Per this and the Playwright config, I think that you, @BRKalow, and I should connect and talk this through.

In this particular instance only three packages were added, we can segment out what we want to install (if we want), and it makes development infinitely easier. Though, this is probably a lot better to show in-person.

Reply:
After scrolling through the PR and leaving my https://github.com/clerk/javascript/pull/2306#discussion_r1423801772 comment I'm even more inclined to not do this. I don't think we need that playground (inside the repo).

Reply:
I completely agree with @LekoArts here.

I've raised this before, but I also think we should remove all playground apps from the repo. People working on a feature can share a playground by using a second repository and cloning it in `playground/` without checking it in the `javascript` repo itself.

I will bring this up again in the next staff meeting.